SensusService/Probes/Probe.cs

```diff
@@ -62,6 +62,7 @@ public static List<Probe> GetAll()
         private List<DateTime> _successfulHealthTestTimes;
         private List<ChartDataPoint> _chartData;
         private int _maxChartDataCount;
+        private bool _runLocalCommitOnStore;
```

Comment:
Sorry, Wes's stuff. I didn't realize some o f his changes had been mixed in with mine.